### PR TITLE
Manta Staking Parameters

### PR DESCRIPTION
**Pull request by Garandor**
- Split manta tx fees 10/90 between author&treasury ( no burn on manta )
- set 3% annual staking inflation
- reenable staking integration tests
- [ ] ~~Ensure manta collators have >=400k $MANTA stake~~ postponed to #1114 

Signed-off-by: Adam Reif <Garandor@manta.network>

## Description

---

Before we can approve this PR for merge, please make sure that **all** the following items have been checked off:
- [x] Connected to an issue with discussion and accepted design using zenhub "Connect issue" button below
- [x] Added **one** label out of the `L-` group to this PR
- [x] Added **one or more** labels from the `A-` and `C-` groups to this PR
- [x] Explicitly labelled `A-calamari` and/or `A-manta` if your changes are meant for/impact either of these (CI depends on it)
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

Situational Notes:
- If adding functionality, write unit tests!
- If importing a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- If needed, update our Javascript/Typescript APIs. These APIs are officially used by exchanges or community developers.
- If modifying existing runtime storage items, make sure to implement storage migrations for the runtime and test them with `try-runtime`. This includes migrations inherited from upstream changes, and you can search the diffs for modifications of `#[pallet::storage]` items to check for any.

### CODE REVIEW

File: runtime/integration-tests/src/integrations_mock/test_common.rs [hunk 1]
```diff
@@ -54,12 +54,21 @@ use super::{ALICE, ALICE_SESSION_KEYS};
 use sp_core::sr25519;
 use sp_runtime::{DispatchError, ModuleError};
 
-// currently, we ignore all parachain staking tests in integration tests
 mod parachain_staking_tests {
     use super::*;
+    cfg_if::cfg_if! {
+            if #[cfg(feature = "calamari")] {
```
Comment (ferrell-code) [inline runtime/integration-tests/src/integrations_mock/test_common.rs]: fmt

Comment (ferrell-code) [reply runtime/integration-tests/src/integrations_mock/test_common.rs]: Also could just add to the giga import in `mod.rs` this is fine too

**Review by ferrell-code — COMMENTED** [review]
nit

**Review by ghzlatarev — DISMISSED** [review]

File: runtime/integration-tests/src/integrations_mock/test_common.rs (hunk 1 above)
Comment (Garandor) [reply runtime/integration-tests/src/integrations_mock/test_common.rs]: fixed, thx

**Review by ghzlatarev — DISMISSED** [review]

File: runtime/integration-tests/src/integrations_mock/test_common.rs (hunk 1 above)
Comment (Garandor) [reply runtime/integration-tests/src/integrations_mock/test_common.rs]: good point thx, turns out the types were actually already there 😅 
Fixed

**Review by ghzlatarev — DISMISSED** [review]

**Review by ghzlatarev — APPROVED** [review]

**Review by ferrell-code — APPROVED** [review]

**Pull request merged**